lib/seahorn/HornifyModule.cc

```diff
@@ -279,12 +319,40 @@ bool HornifyModule::runOnModule(Module &M) {
       Changed = (runOnFunction(*f) || Changed);
   }
 
+  LOG(
+      "inter_mem_counters", if (InterProcMem) {
+        outs() << "BRUNCH_STAT " << "NumParams" << " " << m_n_params << "\n";
+        outs() << "BRUNCH_STAT " << "NumCallSites" << " " <<  m_n_callsites << "\n";
+        outs() << "BRUNCH_STAT " << "NumGlobalV"
+               << " " << m_n_gv << "\n";
+        outs() << "BRUNCH_STAT " << "NumCopiedBytes"
+               << " " << m_fields_copied << "\n";
+        outs() << "BRUNCH_STAT " << "NumCopiedParams"
+               << " " <<  m_params_copied << "\n";
+        outs() << "BRUNCH_STAT " << "NumCopiedGlobalV"
+               << " " << m_gv_copied << "\n";
+        outs() << "BRUNCH_STAT "  << "NumCopiedCallSites"
+               << " " << m_callsites_copied << "\n";
+        outs() << "BRUNCH_STAT " << "NumCalleeArrayNodes"
+               << " " << m_node_array << "\n";
+        outs() << "BRUNCH_STAT " << "NumCalleeOffsetCollapsedNodes"
+               << " " << m_node_ocollapsed << "\n";
+        outs() << "BRUNCH_STAT " << "NumCalleeUnsafeNodes"
+               << " " << m_node_unsafe << "\n";
+      });
+
   if (!m_db.hasQuery()) {
     // --- This may happen if the exit block of main is unreachable
     //     but still the main function can fail.
     m_db.addQuery(mk<TRUE>(m_efac));
   }
 
+  // DEBUG: printing clauses
+  LOG("print_clauses", errs() << "------- PRINTING CLAUSE DB ------\n";);
```

Comment:
Use one `LOG` statement with many instructions instead of multiple `LOG`

Reply:
ok!